framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -402,7 +405,7 @@ public void run() {
   private static void runGenerateScript(DataSource datasource) {
     int exitCode = 0;
     String command = CWD + "/resources/" + datasource.src;
-    LOG.debug("Running command " + command);
+    LOG.info("Running command " + command);
```

Comment:
Just wanted to add that I also think we should display what scripts are running. Having more information would help. If this needs to be part of debug, then our Jenkins scripts need to be updated to run in debug mode by default.